umbrella/MAINTAIN.md

```diff
@@ -0,0 +1,7 @@
+`polkadot-sdk` should be updated when another crate is added to or removed from the workspace. There is a CI check to ensure that it is in sync with the [polkadot-sdk](https://github.com/paritytech/polkadot-sdk) workspace.​
```

Comment:
nit: we can also mention there is a CI check that ensures `polkdaot-sdk` is not missing crates.